packages/schema/src/config/app.ts

```diff
@@ -152,6 +152,16 @@ export default defineUntypedSchema({
      */
     pageTransition: false,
 
+    /**
+     * Default values for view transitions.
+     *
+     * This can be overridden with `definePageMeta` on an individual page.
+     * Only booleans are allowed.
+     * @see https://nuxt.com/docs/getting-started/transitions#view-transitions-api-experimental
```

Comment:
```suggestion
     * This can be overridden with `definePageMeta` on an individual page.
     * @see https://nuxt.com/docs/getting-started/transitions#view-transitions-api-experimental
```